Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -90,6 +90,20 @@ def run
           end
         end
 
+        # Skip packages that are autobumped by BrewTestBot, if there are any.
+        if autobump_core_path.exist? && autobump_cask_path.exist?
+          autobump_core = File.read(autobump_core_path).lines.map(&:strip)
+          autobump_cask = File.read(autobump_cask_path).lines.map(&:strip)
```

Comment:
```suggestion
          autobump_core = autobump_core_path.read.lines.map(&:strip)
          autobump_cask = autobump_cask_path.read.lines.map(&:strip)
```
or `.readlines` may make sense